css/style.css

```diff
@@ -288,5 +288,44 @@ footer p {
   font-weight: 600;
 }
 
+#hamburger {
+  display: none;
+}
+
+/*MEDIA QUERIES*/
+
+@media screen and (max-width: 768px){
+body {
+  margin: 0;
+  padding: 1rem;
```

Comment:
You don't want to apply padding to the entire body in the smaller view, because the background images won't take up the full width of your screen.